From fb6d0149a223b7d3226ddf7e03ab60f172a2ee2a Mon Sep 17 00:00:00 2001
From: "Ooi, Joyce" <joyce.ooi@intel.com>
Date: Fri, 29 Nov 2019 23:26:42 +0800
Subject: [PATCH 081/120] MAINTAINERS: Add entry for Intel FPGA Quad Speed
 Ethernet Driver

commit 1b2250fae96f5499012048091e9ad8e62271c1ea from
https://github.com/altera-opensource/linux-socfpga.git

Add a MAINTAINERS entry covering the Intel FPGA Quad Speed Ethernet Driver

Signed-off-by: Ooi, Joyce <joyce.ooi@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 MAINTAINERS | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index f5e7745c6d4b..fe78de56f9c3 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7942,6 +7942,13 @@ F:	drivers/net/ethernet/intel/
 F:	drivers/net/ethernet/intel/*/
 F:	include/linux/avf/virtchnl.h
 
+INTEL FPGA QUAD SPEED ETHERNET DRIVER
+M:      Joyce Ooi <joyce.ooi@intel.com>
+L:      netdev@vger.kernel.org
+S:      Maintained
+F:      Documentation/devicetree/bindings/net/intel_fpga_qse.txt
+F:      drivers/net/ethernet/altera/intel_fpga_qse_ll
+
 INTEL FRAMEBUFFER DRIVER (excluding 810 and 815)
 M:	Maik Broemme <mbroemme@libmpq.org>
 L:	linux-fbdev@vger.kernel.org
-- 
2.31.1

